lib/edip/options.ex

```diff
@@ -4,33 +4,67 @@ defmodule Edip.Options.Package do
             prefix: nil
 end
 
+defmodule Edip.Options.Mapping do
+  defstruct from: nil,
+            to:   nil
+end
+
 defmodule Edip.Options do
   alias Edip.Options.Package
+  alias Edip.Options.Mapping
 
-  @option_aliases [n: :name, t: :tag, p: :prefix, s: :silent]
-  @option_stricts [name: :string, tag: :string, prefix: :string, silent: :boolean]
+  @option_aliases [n: :name, t: :tag, p: :prefix, s: :silent, m: :mapping]
+  @option_stricts [
+    name:    :string,
+    tag:     :string,
+    prefix:  :string,
+    silent:  :boolean,
+    mapping: [:string, :keep]
+  ]
 
-  defstruct silent:  false,
-            writer:  &Edip.Utils.PrefixWriter.write/1,
-            package: %Package{}
+  defstruct silent:   false,
+            writer:   &Edip.Utils.PrefixWriter.write/1,
+            package:  %Package{},
+            mappings: []
 
   def from_args(args) do
     valid_options = parse_args(args)
 
     writer  = writer(valid_options)
     package = struct(Package, valid_options)
-    struct(__MODULE__, Dict.merge(valid_options, [writer: writer, package: package]))
+    mappings = valid_options[:mappings]
+    struct(__MODULE__, Dict.merge(valid_options, [writer: writer, package: package, mappings: mappings]))
   end
 
-  defp parse_args(args) do
-    {valid_opts, _, _} = OptionParser.parse(args, aliases: @option_aliases, strict: @option_stricts)
-    valid_opts
+  def parse_args(args) do
+    {parsed_opts, _, _} = OptionParser.parse(args, aliases: @option_aliases, strict: @option_stricts)
+    |> IO.inspect
+    |> consolidate_mappings
+
+    parsed_opts
   end
 
-  defp writer(valid_options) do
+  def writer(valid_options) do
     case Dict.get(valid_options, :silent) do
       true -> &Edip.Utils.LogWriter.write/1
       _    -> &Edip.Utils.PrefixWriter.write/1
     end
   end
+
+  def consolidate_mappings({parsed, argv, errors}) do
+    mappings = parsed
+    |> Keyword.get_values(:mapping)
+    |> Enum.map(&create_valid_mapping/1)
+
+    parsed = parsed |> Keyword.delete(:mapping) |> Keyword.merge(mappings: mappings)
+
+    {parsed, argv, errors}
+  end
+
+  def create_valid_mapping(mapping_opt) do
+    case mapping_opt |> String.split(":") do
```

Comment:
There can be actually 2 `:` in a mapping:

```
from:to:access_options

access_options = rw | ro | ...
```

We should take this into account and respect such options, too.
Would you update and extend this?